resolwe_bio/management/commands/generate_etc.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import csv
+import gzip
+import json
+import os
+import random
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.utils import timezone
+from resolwe.flow.models import Data, DescriptorSchema, Process, Storage
+from resolwe_bio.tools.utils import escape_mongokey, gzopen
+
+
+class Command(BaseCommand):
+
+    """Generate ETC objects"""
+
+    help = "Generate ETC objects"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-e', '--n-etc', type=int, default=3,
+                            help="Number of ETC objects to generate (default: %(default)s)")
+        parser.add_argument('--rseed', action='store_true', help="Use fixed random seed")
+
+
+    def set_user(self):
+        users = User.objects.filter(is_superuser=True).order_by('date_joined')
```

Comment:
Use `from django.contrib.auth import get_user_model` and then `get_user_model().objects...`.

This will work even if other user model is used.